doc/schema_template.md

```diff
@@ -10,11 +10,9 @@ If you find a bug please report it [here](https://github.com/galaxyproject/galax
 
 This document serves as reference documentation. If you would like to learn
 how to build tools for Galaxy,
-[Planemo](http://planemo.readthedocs.io/en/latest/writing.html) features a
+[Planemo](https://planemo.readthedocs.io/en/latest/writing.html) features a
 number of tutorials on building Galaxy tools that would better serve that purpose.
 
-$toc
```

Comment:
Found the reason why Sphinx is ignoring it: if `depth` is a float, Sphinx silently skips the toc creation. The above RST command should have been:
```rst
.. contents::
  :depth: 2
..
```
According to https://pandoc.org/changelog.txt , this has been fixed in Pandoc 1.14:
```
* RST writer:

    + Fixed toc depth in RST writer.  Previously the depth was being
      rendered as a floating point number with a decimal point.
```

I'll prepare a workaround.